src/admin/controller/modules/admin-ops.xqy

```diff
@@ -9,11 +9,19 @@ declare function admin-ops:document-insert(
 )
 as empty-sequence()
 {
-  xdmp:document-insert(
-    $uri,
-    $doc,
-    xdmp:default-permissions()
-  )
+  if (fn:doc-available($uri)) then
```

Comment:
If the document doesn't exist, we want to apply the user's default collections rather than the empty sequence. You get the default ones by not passing anything.